compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -417,6 +417,16 @@ def AMDAIELoweringStrategy :
                    "Use the IREE lowering to objectFifos"),
         clEnumValN(mlir::iree_compiler::AMDAIE::LowerToAIEPassPipeline::AIR, "air",
                    "Use the IREE lowering through AIR")
+      )}]>,
+    Option<"targetDevice", "target-device",
+      "mlir::iree_compiler::AMDAIE::AMDAIEDevice",
+      /*default=*/"mlir::iree_compiler::AMDAIE::AMDAIEDevice::npu1_4col",
+      "AIE device to target",
+      [{::llvm::cl::values(
+        clEnumValN(mlir::iree_compiler::AMDAIE::AMDAIEDevice::npu1_4col, "phoenix",
+                   "Compile for Phoenix"),
+        clEnumValN(mlir::iree_compiler::AMDAIE::AMDAIEDevice::npu4, "strix",
+                   "Compile for Strix")
```

Comment:
I think we should  use `NPU` naming: `npu1_4col`, `npu4`, instead of phoenix/strix, for consistency and to support multiple versions of a device generation. For example, while we're not targeting the 4 column version, there exists a strix 4 column version next to the 8 column one.

Reply:
https://github.com/amd/xdna-driver/blob/ca8ecbf43bb62ff6e44185e5d6a34846e3ea1a7e/src/driver/amdxdna/amdxdna_pci_drv.c#L47